### PR TITLE
Add support for controller names without either .directive or .controller in the file name

### PR DESCRIPTION
**Pull request by pums12**

### CODE REVIEW

File: Rosieks.VisualStudio.Angular/Commands/GoToCodeCmd.cs [hunk 1]
```diff
@@ -92,9 +94,14 @@ private void MenuItemCallback(object sender, EventArgs e)
         private string GetCodePath(string path)
         {
             var fileName = Path.GetFileNameWithoutExtension(path);
-            fileName = fileName.Contains(".directive.") ? fileName : fileName + ".controller";
-            string codePath = Path.Combine(Path.GetDirectoryName(path), fileName);
-            if (FileHelper.TryFind(codePath, AngularPackage.CodeExtensions, out codePath))
+            var searchPaths = new List<string>();
+
+            var adjustedFileName = fileName.Contains(".directive.") ? fileName : fileName + ".controller";
```
Comment (rosieks) [inline Rosieks.VisualStudio.Angular/Commands/GoToCodeCmd.cs]: If the original `fileName` is for example _menu.directive.js_ then you will search for such files:
* `adjustedFileName` -> _menu.directive.js_
* `fileName` -> _menu.directive.js_

And as I understood you would like to look for _menu.js_

File: Rosieks.VisualStudio.Angular/Commands/GoToCodeCmd.cs [hunk 2]
```diff
@@ -1,4 +1,6 @@
-﻿namespace Rosieks.VisualStudio.Angular.Commands
+﻿using System.Collections.Generic;
```
Comment (rosieks) [inline Rosieks.VisualStudio.Angular/Commands/GoToCodeCmd.cs]: Please move that into the namespace like other usings.

File: Rosieks.VisualStudio.Angular/Extensions/FileHelper.cs [hunk 3]
```diff
@@ -3,6 +3,17 @@
     using System.IO;
     internal class FileHelper
     {
+        internal static bool TryFind(string[] paths, string[] extensions, out string fileName) {
```
Comment (rosieks) [inline Rosieks.VisualStudio.Angular/Extensions/FileHelper.cs]: Please move all `{` to next line.

**Comment by pums12** [comment]
I am preparing another Pull Request to tidy up the code as requested.

Yes, I would also like to search for menu.js

I have inherited a project with the following naming structure:

View: {viewName}.cshtml
Controller: {viewName}.js

**Pull request closed without merging**